streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -1168,7 +1169,32 @@ public void kGroupedStreamZeroArgCountShouldPreserveTopologyStructure() {
 
     @Test
     public void kGroupedStreamNamedMaterializedCountShouldPreserveTopologyStructure() {
-        final StreamsBuilder builder = new StreamsBuilder();
+        Properties streamsProp = new Properties();
+        streamsProp.put(StreamsConfig.APPLICATION_ID_CONFIG, "streams-config-test");
+        streamsProp.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
+        final StreamsBuilder builder = new StreamsBuilder(streamsProp);
+        builder.stream("input-topic")
+            .groupByKey()
+            .count(Materialized.as("count-store"));
+        final TopologyDescription describe = builder.build().describe();
```

Comment:
An example we put the `stream props` into `StreamsBuilder` constructor and build with default store implementation setting for `KeyValueBytesStoreSupplier`. And the same for `WindowBytesStoreSupplier` and `SessionBytesStoreSupplier` below